_pages/fr/index.html

```diff
@@ -16,55 +16,92 @@ <h1 property="name" id="wb-cont">{{ site.data.i18n.general.head.title.fr }}</h1>
+  </div>
 
-      <div class="col-md-6">
-        <section>
-          <h3><a href="{{ site.data.i18n.general.nav.software.link[page.lang] }}">{{ site.data.i18n.general.nav.software[page.lang] }}</a></h3>
-          <p>Effectuez des recherches dans les logiciels libres utilisés par les administrations publiques Canadiennes, promouvez la collaboration et la réutilisation de solutions préférées.</p>
-        </section>
-      </div>
+  <div class="col-md-4">
+    <section class="panel panel-info">
 
-      <div class="col-md-6">
-        <section>
-          <h3><a href="{{ site.data.i18n.general.nav.standard.link[page.lang] }}">{{ site.data.i18n.general.nav.standard[page.lang] }}</a></h3>
-          <p>Effectuez des recherches dans les normes ouvertes utilisés par les administrations publiques Canadiennes.</p>
-        </section>
-      </div>
+      <header class="panel-heading">
+        <a href="{{ site.data.i18n.general.nav.code.link[page.lang] }}" style="color: inherit !important; text-decoration: none !important"><h5 class="panel-title">Code source ouvert</h5></a>
+      </header>
 
-      <div class="col-md-6">
-        <section>
-          <h3><a href="{{ site.data.i18n.general.nav.doc.link[page.lang] }}">{{ site.data.i18n.general.nav.doc[page.lang] }}</a></h3>
-          <p>Découvrez comment contribuer en utilisant un schéma de métadonnées standard pour chaque service.</p>
-        </section>
+      <div class="panel-body" style="padding-bottom: 62px !important">
+        <div class="col-md-3" style="color: #31708f; font-size: 50px">
+          <a href="{{ site.data.i18n.general.nav.code.link[page.lang] }}" style="color: inherit !important; text-decoration: none !important"><i class="fas fa-code"></i></a>
+        </div>
+        <div class="col-md-9">
+          <p>Recherchez des projets source ouverts et des répertoires de code.</p>
+        </div>
       </div>
 
-    </div>
-  </section>
+    </section>
+  </div>
+
+  <div class="col-md-4">
+    <section class="panel panel-success">
+      <header class="panel-heading">
+        <a href="{{ site.data.i18n.general.nav.software.link[page.lang] }}" style="color: inherit !important; text-decoration: none !important"><h5 class="panel-title">Logiciels libres</h5></a>
+      </header>
+
+      <div class="panel-body" style="padding-bottom: 0px !important">
+        <div class="col-md-3" style="color: #3c763d; font-size: 50px">
+          <a href="{{ site.data.i18n.general.nav.software.link[page.lang] }}" style="color: inherit !important; text-decoration: none !important"><i class="fa fa-cog"></i></a>
+        </div>
+        <div class="col-md-9">
+          <p>Recherchez les plateformes de logiciels libres utilisés par les administrateurs publiques canadiens.</p>
+        </div>
+      </div>
 
-  <div class="col-md-4 col-xs-12 pull-right">
-    <section class="lnkbx">
-      <h2>Derniers codes source ouverts</h2>
-      {% include codeLastUpdated.html %}
```

Comment:
This is the only time the codeLastUpdated include is reference. Consider removing the include file altogether.

Reply:
Also in the en version of the page

Reply:
has been removed